src/KomaMRI.jl

```diff
@@ -2,12 +2,9 @@ module KomaMRI
 
 using Reexport
 
-# MRIReco
-using MRIReco
-@reexport using MRIReco: reconstruction
-
 # IMPORT PACKAGES
 using Reexport
+@reexport using KomaMRIBase
 @reexport using KomaMRICore
```

Comment:
KomaMRICore also exports KomaMRIBase?

Reply:
KomaMRICore uses using with KomaMRIBase without exporting its functions. Should we export the functions from KomaMRIBase when using KomaMRICore? (I believe the answer is yes.)

Reply:
Maybe you are not doing it with `@reexport` but you are exporting (in Core): ``

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/240/commits/5627a6172d4ddbb239406d81849c3102ebda1a85